pages/api/users/index.js

```diff
@@ -0,0 +1,23 @@
+import dbConnect from "@/db/connect";
+import User from "@/db/models/User";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  try {
+    switch (request.method) {
+      case "GET": {
+        const user = await User.find();
```

Comment:
```suggestion
        const users = await User.find();
```

or even better: Try to find the specific user. If you cant find one sende an error 404.

Reply:
insert 404 for all route with method GET